internal/task/task.go

```diff
@@ -0,0 +1,142 @@
+package task
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/ivanzharinov/go_final_project/internal/utils"
+	"io/ioutil"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/ivanzharinov/go_final_project/internal/db"
+)
+
+type AddTaskRequest struct {
+	Date    string `json:"date"`
+	Title   string `json:"title"`
+	Comment string `json:"comment"`
+	Repeat  string `json:"repeat"`
+}
+
+type AddTaskResponse struct {
+	ID    string `json:"id,omitempty"`
+	Error string `json:"error,omitempty"`
+}
+
+func AddTask(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+
+	body, err := ioutil.ReadAll(r.Body)
```

Comment:
Типо функция уже deprecated
```suggestion
	body, err := io.ReadAll(r.Body)
```

Reply:
исправлено